Zend/zend_vm_execute.h

```diff
@@ -4491,7 +4491,7 @@ static ZEND_VM_COLD ZEND_OPCODE_HANDLER_RET ZEND_FASTCALL ZEND_RETURN_BY_REF_SPE
 			break;
 		}
 
-		retval_ptr = NULL;
+		retval_ptr = (zend_unreachable(), NULL);
```

Comment:
Don't use this magic syntax. Try to use something like this
```c
retval_ptr = zend_get_bad_ptr();
...
static zend_always_inline void* zend_get_bad_ptr(void)
{
	ZEND_UNREACHABLE();
	return NULL;
}
```

Reply:
It *is* well defined ("Comma operator" in the C spec) but it's true that many people are probably not aware of it. I'll go for your solution.

Reply:
or
```c
retval_ptr = BAD_PTR
...
#define BAD_PTR zend_get_bad_ptr()
static zend_always_inline void* zend_get_bad_ptr(void)
{
	ZEND_UNREACHABLE();
	return NULL;
}
```